src/javascripts/ng-admin/Main/component/directive/maNgAdmin.js

```diff
@@ -0,0 +1,18 @@
+function maNgAdminDirective(NgAdminConfiguration) {
+    'use strict';
+
+    return {
+        scope: {
+            'configuration': '&'
```

Comment:
If it's hard, it may be because it's not possible, see http://stackoverflow.com/questions/22313492/angular-ui-router-get-two-states-active-in-parallel. Bummer.